CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Changed
+
+- Migrate filelog operators to follow opentelemetry-log-collection v0.29.0 changes ([Migrate filelog operators to follow v0.29.0 changes #436](https://github.com/signalfx/splunk-otel-collector-chart/pull/436))
+  - [BREAKING CHANGE] Several breaking changes were made that affect the
```

Comment:
It may be a good time to add an unreleased section to https://github.com/signalfx/splunk-otel-collector-chart/blob/main/UPGRADING.md so we can associate some updated examples with these changes.

Reply:
To move forward, I'd like to merge this PR the way it is. I'll open up a new PR today for updating UPGRADING.md.